internal/api/authz/permissions_test.go

```diff
@@ -64,7 +64,7 @@ func Test_GetUserMethodPermissions(t *testing.T) {
 		{
 			name: "Empty Context",
 			args: args{
-				ctx: getTestCtx("", ""),
+				ctxData: GetCtxData(getTestCtx("", "")),
```

Comment:
please simplfy this:
getTestCtx writes the contextData into context and GetCtxData retrieves it from it

Reply:
and the ones below